pingcap/tiflash
# Add debug image build target for CentOS7

### What problem does this PR solve?

Issue Number: close #5343 
BTW, add debug build script for Darwin.

Problem Summary:

### What is changed and how it works?

### Check List

Tests 


1. Clone tiflash code, and enable any one of the existing Failpoints and trigger it in "#ifdef FIU_ENABLE" block of Server::main
2. In CentOS7, cd ${TiFlash_Root}/release-centos7-llvm/
3. sudo make build_tiflash_debug_amd64
4. Run "tiup playground nightly --tiflash.binpath ${TiFlash_Root}/release-centos7-llvm/tiflash/tiflash"
5. Check TiFlash server failed to start during to the triggered FailPoint. 

Side effects


Documentation


### Release note



```release-note
None
```
